docs/overview/tezos-different.md

```diff
@@ -20,12 +20,12 @@ Anyone who holds XTZ — the chain's native token — can propose changes to how
 
 ## Tezos uses proof of stake
 
-The proof-of-stake consensus model eliminates the need for high energy use, making it the "green" choice for blockchains. Instead of competing to achieve consensus as in proof-of-work models, Tezos nodes (called *bakers*) stake Tezos tokens to earn the right to create blocks and receive rewards. Users who want to participate without running a node themselves can delegate tokens to a baker for a share of the rewards. The bakers and delegators keep control of their tokens and can remove them at any time. Tezos's approach to consensus has been described as [Liquid Proof of Stake](https://medium.com/tezos/liquid-proof-of-stake-aec2f7ef1da7).
+The proof-of-stake consensus model eliminates the need for high energy use, making it the "green" choice for blockchains. Instead of competing to achieve consensus as in proof-of-work models, Tezos nodes (called *bakers*) stake Tezos tokens to earn the right to create blocks and receive rewards. Users who want to participate without running a node themselves can delegate tokens to a baker for a share of the rewards. The bakers and delegators keep control of their tokens: bakers can vary their staked amount in time, and delegators can spend their tokens or change their delegate at any time. Tezos's approach to consensus has been described as [Liquid Proof of Stake](https://medium.com/tezos/liquid-proof-of-stake-aec2f7ef1da7).
```

Comment:
Even though "can remove them at any time" is not technically accurate, your version doesn't say that bakers can withdraw their tokens at all. How about "The bakers and delegators keep control of their tokens and can withdraw them later." This should be enough at  this level of detail.

Reply:
OK for your version, it removes the mistake, while staying more hihg-level.

Reply:
In fact, let's be a bit more coherent, opposing "stake" with "unstake", instead of "withdraw", and also suggest that the control is not total, but is about unstaking later:
```suggestion
The proof-of-stake consensus model eliminates the need for high energy use, making it the "green" choice for blockchains. Instead of competing to achieve consensus as in proof-of-work models, Tezos nodes (called *bakers*) stake Tezos tokens to earn the right to create blocks and receive rewards. Users who want to participate without running a node themselves can delegate tokens to a baker for a share of the rewards. The bakers and delegators keep control of their tokens in that they can unstake them later. Tezos's approach to consensus has been described as [Liquid Proof of Stake](https://medium.com/tezos/liquid-proof-of-stake-aec2f7ef1da7).
```